config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
i was just trying to format as per the mockup, happy to move/rename/remove if others would prefer.  i also sure that was the right place to put that, that's just what i'd gathered from reading i18n docs and discussion.